src/app/area/area.demo.ts

```diff
@@ -13,7 +13,10 @@ export class AreaDemoComponent implements OnInit {
 
   @ViewChild(SohoLineComponent) sohoLineComponent: SohoLineComponent;
 
-  private selection: SohoLineSelected  = {groupIndex: 0, fieldName: 'name', fieldValue: 'Category B'};
+  // private selection: SohoLineSelected  = {groupIndex: 1};
```

Comment:
remove all the commented lines?

Reply:
I added comments to each file to indicate that the commented out code are alternate examples of how to set the selection on charts.